end_to_end/tpu/test_orbax_to_hf.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# This script is to test the flow of MaxText/llama_mistral_mixtral_orbax_to_hf.py.
+# Steps in the script:
+# 1. Convert MaxText orbax ckpt to HF using MaxText/llama_mistral_mixtral_orbax_to_hf.py
+# 2. Confirm the logits match for MaxText orbax ckpt and the new HF ckpt created in step 2.
+set -ex
+export MODEL_VARIATION='llama3.1-8b'
+
+export BASE_OUTPUT_PATH=gs://runner-maxtext-logs/2024-12-18-17-35
+
+export CONVERTED_CHECKPOINT=${BASE_OUTPUT_PATH}/${MODEL_VARIATION}/scanned_chkpt/0/items
+export RUN_NAME=unscann_llama3.1
+# We defined path to unscanned checkpoint created in 1_test_llama3.1_8b.sh
+export UNSCANNED_CKPT_PATH=${BASE_OUTPUT_PATH}/${RUN_NAME}/checkpoints/0/items
+
+# converting MaxText orbax ckpt to HF
+
+JAX_PLATFORMS=cpu python3 MaxText/llama_mistral_mixtral_orbax_to_hf.py MaxText/configs/base.yml base_output_directory=gs://runner-maxtext-logs \
+    load_parameters_path=gs://runner-maxtext-logs/2024-12-18-17-35/llama3.1-8b/scanned_chkpt/0/items run_name=convert_to_hf \
+    model_name=llama3.1-8b hf_model_path=/home/mohitkhatwani/maxtext/hf_llama3.1_new/
+
+
+python MaxText/scratch_code/golden_llama3-70b_export.py --model-id /home/mohitkhatwani/maxtext/hf_llama3.1_new/ --output-path golden_data_new_llama3_1_8b.jsonl
+
+pushd MaxText/test_assets
+gcloud storage cp gs://maxtext-llama/Llama3_1_8B/golden-logits/golden_data_new_llama3_1_8b.jsonl .
+popd
+
+# comparing logits of the HF ckpt above
+
+python3 MaxText/tests/forward_pass_logit_checker.py MaxText/configs/base.yml base_output_directory=${BASE_OUTPUT_PATH} \
```

Comment:
It is best to load the converted model in HF and ensure the logits match. 

Please see an example [here](https://github.com/AI-Hypercomputer/maxtext/blob/fc32a58ee6f0278480659a172e67a2963b599565/MaxText/tests/orbax_to_hf_logit_checker.py#L97)